SECURITY_GUIDANCE.md

```diff
@@ -30,11 +30,36 @@ However, it is always better to patch vulnerabilities than to rely on mitigation
 We provide [a Kubernetes operator](https://github.com/bottlerocket-os/bottlerocket-update-operator) for automated updates to Bottlerocket.
 We recommend deploying it on your Kubernetes clusters.
 
+### Avoid containers with elevated privileges
+
+Containers can be made more secure by limiting the capabilities they have, by filtering syscalls they can make, and by changing the SELinux labels they use.
+
+Capabilities are a way to split up the traditional powers of the `root` user so that a subset of the permissions can be granted instead.
+For example, `CAP_NET_BIND_SERVICE` can be granted to allow binding to a low-numbered port.
+Bottlerocket uses `runc` to execute containers with [a subset of Linux capabilities](https://github.com/opencontainers/runc/blob/master/libcontainer/SPEC.md#security).
+
+Syscalls are a way for userspace programs to request services from the kernel.
+Seccomp filters can be used to allow access to a subset of syscalls.
+Bottlerocket uses `containerd` as the container runtime which provides [a default seccomp profile](https://github.com/containerd/containerd/blob/master/contrib/seccomp/seccomp_default.go).
+
+SELinux labels are part of mandatory access controls, which impose constraints after discretionary access controls are checked.
+Bottlerocket runs all containers with the unprivileged `container_t` label today.
+However, privileged containers may run with the privileged `super_t` label in the future.
+
+Orchestrators provide ways to disable these protections.
+For example, Docker can run containers with the `--privileged` flag, and Kubernetes can run pods with `privileged: true` set in the pod definition.
```

Comment:
Is it worth adding something about ECS here as well?  ECS allows containers to be run with a `privileged` flag in the task definition.